src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+Title: Basic client usage - tutorial
+TODO:  Warning: Ubuntu release versions hardcoded
+       Include links to planned tutorials (tutorial-bundles.md, tutorial-constraints.md)
+
+# Basic client usage - tutorial
+
+The goal of this tutorial is to give new Juju operators a solid introduction to
+the command line client. In so doing, several important concepts are
+introduced. It also discusses the use of command prefixes and aliases. Further
+reading suggestions are included at the end.
+
+## Prerequisites
+
+The following prerequisites are assumed as a starting point for this tutorial:
+
+ - You're using Ubuntu 18.04 LTS (Bionic).
+ - Juju (stable snap channel) is installed. See the [Installing Juju][install]
+   page.
+ - You have chosen a backing cloud and have created a controller for it. Refer
+   to the [Clouds][clouds] page to get started with a cloud and controller.
+ 
+This guide uses LXD (`v.3.9`) as a backing cloud due to its accessibility and
```

Comment:
s/backing//

Reply:
Ditto.

The reason why "backing cloud" was introduced (years ago) was that just "cloud" was considered too vague. A backing cloud is a cloud that is being used *with* Juju. "Your cloud" just sounds, again, vague in many contexts.